Demo/build.gradle

```diff
@@ -61,6 +61,31 @@ android {
             java.srcDirs += "src/androidTestShared/java"
         }
     }
+
+    // https://developer.android.com/build/build-variants#groovy
+    signingConfigs {
+        release {
+            storeFile file(System.getenv('DEMO_KEYSTORE_FILE') ?: 'debug.keystore')
+            storePassword System.getenv('DEMO_KEYSTORE_PASSWORD') ?: 'android'
+            keyAlias System.getenv('DEMO_KEY_ALIAS') ?: 'androiddebugkey'
+            keyPassword System.getenv('DEMO_KEY_PASSWORD') ?: 'android'
```

Comment:
Do these variables need to be set locally in order to do a command-line deploy?

Reply:
They do need to be set yes, but they'll be set in the CI environment. I did some manual deploys early on but it should all be automated going forward.